docs/source/programming/vscode-with-slurm-job.md

```diff
@@ -0,0 +1,71 @@
+# Using VSCode with SLURM-managed jobs on the SWC HPC cluster
+
+This guide explains how to set up and use VSCode within a SLURM-managed job on the SWC HPC cluster, offering a solution for users who require fast access to shared storage or substantial computational resources.
+
+This solution is easy to set up and constrained by the SLURM job's resource limits, ensuring that users can work efficiently without overloading the system.
```

Comment:
Well I think the point of doing all this is that if SLURM is restrictive, so will be anything that we run with VSCode tunnel. If we remove this point, we loose the advantage of using code-tunnel instead of ssh-ing into a node.
I can make it more explicit in the sense that is constrained as long as SLURM constrains 😓

Reply:
Also, time limits are there, and my programs still get killed by slurm if I don't ask for enough memory. I am unsure why with numpy array they didn't get killed.

Reply:
aah ok I see! thanks for checking this